qa/rpc-tests/test_framework/authproxy.py

```diff
@@ -132,14 +136,16 @@ def _request(self, method, path, postdata):
             self.__conn.request(method, path, postdata, headers)
             return self._get_response()
 
-    def __call__(self, *args):
+    def __call__(self, *args, **argsn):
         AuthServiceProxy.__id_count += 1
 
         log.debug("-%s-> %s %s"%(AuthServiceProxy.__id_count, self._service_name,
                                  json.dumps(args, default=EncodeDecimal, ensure_ascii=self.ensure_ascii)))
+        if args and argsn:
+            raise ValueError('Cannot handle both named and positional arguments')
         postdata = json.dumps({'version': '1.1',
```

Comment:
In the OP you are linking to the "JSON-RPC 2.0 Specification", which states
>jsonrpc
    A String specifying the version of the JSON-RPC protocol. MUST be exactly "2.0".

Can you explain why you keep this at 1.1?